src/test/java/nah/NahTest.java

```diff
@@ -1,2 +1,57 @@
-package nah;public class NahTest {
+package nah;
```

Comment:
I like how extensive and comprehensive your JUnit tests are. Keep up the good work!